package.json

```diff
@@ -6,8 +6,11 @@
   "scripts": {
     "dev": "ts-node-dev --respawn --pretty --transpile-only src/server.ts",
     "testLocal": "dotenv -e .env.test npm run test",
-    "test": "mocha --file ./test/helper.ts --require ts-node/register 'test/**/*-test.ts'",
-    "start": "tsc && node dist/server.js"
+    "test": "mocha --file ./test/helper.ts --require ts-node/register test/**/*-test.ts",
```

Comment:
you've removed the ' quotes around the 'test/**/*-test.ts' and it is no longer properly running all tests. Your most recent github actions job only ran two tests.